src/ops/MappingOps.ts

```diff
@@ -479,6 +480,31 @@ export function writeSyncJsonToDirectory(
   );
 }
 
+/**
+ * Helper that writes mappings in a managed.idm.json config entity to a directory
+ * @param managed The managed.idm.json config entity
+ * @param directory The directory to save the mappings
+ */
+export function writeManagedJsonToDirectory(
```

Comment:
This function would be better placed in IdmOps, since it doesn't have anything to do with mappings.

Reply:
I made those changes, let me know what you think.